packages/commerce-sdk-react/package.json

```diff
@@ -40,7 +40,7 @@
     "version": "node ./scripts/version.js"
   },
   "dependencies": {
-    "commerce-sdk-isomorphic": "^1.13.1",
+    "commerce-sdk-isomorphic": "^2.0.0",
```

Comment:
Did we inadvertently force a breaking on this project by pumping this dependency as we are exposing the api clients via the `useCommerceApi` hook?

Reply:
yes i think so

Reply:
i'm bumping the major version for commerce-sdk-react too